content/pages/006_languages/python.md

```diff
@@ -0,0 +1,39 @@
+Title: Python Conventions
+Lang: en
+Category: language-specific conventions
+Slug: python-conventions
+Summary: Python conventions
+Status: published
+
+## Python
+* [Python](https://www.python.org/)
+
+## Style Guides
+* [Google style guide](https://google.github.io/styleguide/pyguide.html) 
+
+## Package Recommendations
+* Python version management
+    * [pyenv](https://github.com/pyenv/pyenv)
+    * [anaconda](https://www.anaconda.com/)
+* Package management
+    * [pipenv](https://pipenv.pypa.io/en/latest/)
+* Testing
+    * [unittest](https://docs.python.org/3/library/unittest.html)
+    * [pytest](https://docs.pytest.org/en/latest/)
+* Code formatting
+    * [black](https://black.readthedocs.io/en/stable/)
+    * [isort](https://pycqa.github.io/isort/)
+    * [flake8](https://flake8.pycqa.org/en/latest/)
+* Linting
+    * [black](https://black.readthedocs.io/en/stable/)
+    * [pylint](https://pylint.org/)
+* HTTP requests
+    * [requests](https://docs.python-requests.org/en/latest/)
+* Math
+    * [numpy](https://numpy.org/)
+* Data
+    * [pandas](https://pandas.pydata.org/)
+    * [pyspark](https://spark.apache.org/docs/latest/api/python/)
+    * [pyarrow](https://arrow.apache.org/docs/python/install.html)
+* Visualization
```

Comment:
Add [seaborn](https://seaborn.pydata.org/index.html) and [plotly](https://plotly.com/python/)